### DISCUSSION TITLE
Code for dark mode changed since FF v119

### DISCUSSION THREAD

**Discussion by JohnyP36** [black7375/Firefox-UI-Fix#808] [Q&A]
Up until now I was using `:root[lwt-default-theme-in-dark-mode="true"]` to specify colors, paddings, etc. which should only be applied if I set the dark theme in Firefox. 
But FF updated to version 119 and those rules don't work anymore.

Can you reproduce this?
If yes, do you know the solution?

**Comment by black7375** [comment]
Your answer is late.
I think there are few alternatives, and we need to think about how to enable it through `-moz-bool-pref`.
